java/DEVELOPER.md

```diff
@@ -155,6 +155,46 @@ Some troubleshooting issues:
 - If build fails because of rust compiler fails, make sure submodules are updated using `git submodule update`.
 - If protobuf 26.0 or earlier is detected, upgrade to the latest protobuf release.
 
+## Running Examples App
```

Comment:
Wouldn't this be useful for users of GLIDE to know? I feel like this is less DEVELOPER.MD material and more README.MD material. I put instructions for running the Scala and Kotlin examples in the README in my PR for that reason, but even if you disagree, I think we should be aligned at the very least.

Reply:
I will like to the examples/java/README.md from the jave/README.md and give a short blurb.  You are correct that it would be useful to have. 
But this level of detail (publishing to local maven) is not necessary in the README.md